run.py

```diff
@@ -12,25 +12,28 @@
 from utils import report_generator
 
 
-def main(gear_context):
+def main(fw, gear_context):
```

Comment:
fw is representing access to the SDK, which is a package, and like other python packages, you wouldn't typically pass them through functions but instead import them for a script that requires their utilities.